Scripts/aarch64_fit_native.py

```diff
@@ -16,6 +16,14 @@
         tuple([0x41, 0xd0d]): "cortex-a77",
         tuple([0x41, 0xd41]): "cortex-a78",
         tuple([0x41, 0xd44]): "cortex-x1",
+        tuple([0x41, 0xd47]):
+            [ ["cortex-a78", "0.0"],
+              ["cortex-a710", "999.0"], # Doesn't exist in clang yet
```

Comment:
Could you add the Clang version you checked against for reference?

Reply:
Resolved.